Library/Sources/SCPlayer.m

```diff
@@ -321,6 +321,8 @@ - (void)setItemByAsset:(AVAsset *)asset {
 }
 
 - (void)setItem:(AVPlayerItem *)item {
+    [self unsetupDisplayLink];
+    [self setupDisplayLink];
```

Comment:
The same previous bug, to prevent black screen bug, when playing a movie preview.